# Study 829 — Global Sovereign-Bond Momentum — the teardown

The Newey-West spread *t*, the buy-and-hold benchmark, the 3,000-draw block-rotation placebo, the three-era robustness cut, the lookback sweep, the costed backtest, and the 20-seed synthetic control.

In [1]:
R = {'start': '2007-01-31', 'end': '2026-06-30', 'n_etfs': 5, 'n_months': 221, 'fingerprint': 'fee295eff13f', 'bh_bps': 24.56, 'bh_ann': 2.95, 'bh_sharpe': 0.399, 'bh_t': 1.88, 'ls_bps': 0.26, 'ls_ann': 0.03, 'ls_vol': 5.99, 'ls_sharpe': 0.005, 'ls_t_nw': 0.02, 'ls_t1s': 0.02, 'ls_hit': 0.548, 'ls_pl_obs': 0.26, 'ls_pl_mean': 7.46, 'ls_pl_sd': 12.47, 'ls_pl_p': 0.7243, 'ls_e1_bps': 4.65, 'ls_e1_t': 0.23, 'ls_e2_bps': -9.8, 'ls_e2_t': -0.91, 'ls_e3_bps': 5.55, 'ls_e3_t': 0.25, 'ls_c5_net': -3.18, 'ls_c5_cost': 3.44, 'ls_c5_netS': -0.064, 'ls_c5_t': -0.27, 'ls_c10_net': -4.48, 'ls_c10_cost': 4.74, 'ls_c10_netS': -0.09, 'ls_c10_t': -0.39, 'ls_turn': 0.26, 'lo_bps': 12.27, 'lo_ann': 1.47, 'lo_vol': 4.56, 'lo_sharpe': 0.323, 'lo_t_nw': 1.75, 'lo_t1s': 1.39, 'lo_hit': 0.516, 'lo_pl_obs': 12.27, 'lo_pl_mean': 13.57, 'lo_pl_sd': 6.92, 'lo_pl_p': 0.533, 'lo_e1_bps': 26.94, 'lo_e1_t': 1.66, 'lo_e2_bps': 8.66, 'lo_e2_t': 0.88, 'lo_e3_bps': 2.25, 'lo_e3_t': 0.29, 'lo_c5_net': 11.59, 'lo_c5_netS': 0.306, 'lo_c5_t': 1.31, 'lo_c10_net': 10.92, 'lo_c10_cost': 1.35, 'lo_c10_netS': 0.288, 'lo_c10_t': 1.24, 'lo_turn': 0.14, 'lb6_t': 1.56, 'lb9_t': 1.23, 'lb12_t': 1.75, 'lb15_t': 1.52, 'null_mean_t': -0.22, 'null_fire': 0, 'planted_mean_t': 13.38, 'planted_sharpe': 2.45, 'planted_fire': 20}

## The headline — long-short vs long-only vs just holding the bonds

A trend edge has to beat *owning the bonds*. It doesn't.

In [2]:
print(f"buy-and-hold : {R['bh_bps']:+.2f} bps/mo  Sharpe {R['bh_sharpe']:.3f}  NW t = {R['bh_t']:+.2f}")
print(f"long-short   : {R['ls_bps']:+.2f} bps/mo  Sharpe {R['ls_sharpe']:.3f}  NW t = {R['ls_t_nw']:+.2f}  (hit {R['ls_hit']:.3f})")
print(f"long-only    : {R['lo_bps']:+.2f} bps/mo  Sharpe {R['lo_sharpe']:.3f}  NW t = {R['lo_t_nw']:+.2f}  (hit {R['lo_hit']:.3f})")
print('  -> long-only UNDER-earns buy-and-hold: the +12 bps is bond beta, not trend alpha')

buy-and-hold : +24.56 bps/mo  Sharpe 0.399  NW t = +1.88
long-short   : +0.26 bps/mo  Sharpe 0.005  NW t = +0.02  (hit 0.548)
long-only    : +12.27 bps/mo  Sharpe 0.323  NW t = +1.75  (hit 0.516)
  -> long-only UNDER-earns buy-and-hold: the +12 bps is bond beta, not trend alpha


## Placebo — block-rotate the returns (3,000 draws)

Break the trend->forward-return link; does the signal beat its own return series' drift?

In [3]:
print(f"long-short: observed {R['ls_pl_obs']:+.2f} bps vs null {R['ls_pl_mean']:+.2f} (sd {R['ls_pl_sd']:.2f}) -> p = {R['ls_pl_p']:.4f}")
print(f"long-only : observed {R['lo_pl_obs']:+.2f} bps vs null {R['lo_pl_mean']:+.2f} (sd {R['lo_pl_sd']:.2f}) -> p = {R['lo_pl_p']:.4f}")
print('  long-only observation sits BELOW its own rotation-null mean -> no trend information')

long-short: observed +0.26 bps vs null +7.46 (sd 12.47) -> p = 0.7243
long-only : observed +12.27 bps vs null +13.57 (sd 6.92) -> p = 0.5330
  long-only observation sits BELOW its own rotation-null mean -> no trend information


## Robustness — three eras and a lookback sweep

In [4]:
print('era          long-short            long-only')
print(f"2007-2014    {R['ls_e1_bps']:+6.2f} (t={R['ls_e1_t']:+.2f})    {R['lo_e1_bps']:+6.2f} (t={R['lo_e1_t']:+.2f})")
print(f"2014-2020    {R['ls_e2_bps']:+6.2f} (t={R['ls_e2_t']:+.2f})    {R['lo_e2_bps']:+6.2f} (t={R['lo_e2_t']:+.2f})")
print(f"2020-2026    {R['ls_e3_bps']:+6.2f} (t={R['ls_e3_t']:+.2f})    {R['lo_e3_bps']:+6.2f} (t={R['lo_e3_t']:+.2f})")
print(f"lookback NW t (long-only): 6m {R['lb6_t']:+.2f}  9m {R['lb9_t']:+.2f}  12m {R['lb12_t']:+.2f}  15m {R['lb15_t']:+.2f}  (never reaches 2)")

era          long-short            long-only
2007-2014     +4.65 (t=+0.23)    +26.94 (t=+1.66)
2014-2020     -9.80 (t=-0.91)     +8.66 (t=+0.88)
2020-2026     +5.55 (t=+0.25)     +2.25 (t=+0.29)
lookback NW t (long-only): 6m +1.56  9m +1.23  12m +1.75  15m +1.52  (never reaches 2)


## The costed backtest — one-way turnover + short borrow

One-way cost x turnover per rebalance; short book pays 75 bps/yr borrow.

In [5]:
print(f"long-short  5 bps: gross {R['ls_bps']:+.2f} -> net {R['ls_c5_net']:+.2f} bps/mo (cost {R['ls_c5_cost']:.2f}, net Sharpe {R['ls_c5_netS']:+.3f}, t={R['ls_c5_t']:+.2f})")
print(f"long-short 10 bps: gross {R['ls_bps']:+.2f} -> net {R['ls_c10_net']:+.2f} bps/mo (cost {R['ls_c10_cost']:.2f}, net Sharpe {R['ls_c10_netS']:+.3f}, t={R['ls_c10_t']:+.2f})")
print(f"long-only   5 bps: gross {R['lo_bps']:+.2f} -> net {R['lo_c5_net']:+.2f} bps/mo (net Sharpe {R['lo_c5_netS']:+.3f}, t={R['lo_c5_t']:+.2f})")
print(f"long-only  10 bps: gross {R['lo_bps']:+.2f} -> net {R['lo_c10_net']:+.2f} bps/mo (cost {R['lo_c10_cost']:.2f}, net Sharpe {R['lo_c10_netS']:+.3f}, t={R['lo_c10_t']:+.2f})")

long-short  5 bps: gross +0.26 -> net -3.18 bps/mo (cost 3.44, net Sharpe -0.064, t=-0.27)
long-short 10 bps: gross +0.26 -> net -4.48 bps/mo (cost 4.74, net Sharpe -0.090, t=-0.39)
long-only   5 bps: gross +12.27 -> net +11.59 bps/mo (net Sharpe +0.306, t=+1.31)
long-only  10 bps: gross +12.27 -> net +10.92 bps/mo (cost 1.35, net Sharpe +0.288, t=+1.24)


## Synthetic positive control — the machinery is unbiased

Live: the detector must NOT fire on a random walk and must recover a planted trend.

In [6]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
import numpy as np
from sovereign_momentum import data, strategy as st
null_t = np.array([st.synthetic_detect(data.synthetic_panel(edge=0.0, seed=829+s))['t_nw'] for s in range(8)])
print(f"null (edge=0), 8 seeds: NW t mean {null_t.mean():+.2f} (sd {null_t.std(ddof=1):.2f}), |t|>=2 in {(abs(null_t)>=2).sum()}/8")
planted = st.synthetic_detect(data.synthetic_panel(edge=0.010, seed=829))
print(f"planted (edge=0.010): NW t = {planted['t_nw']:+.2f}, Sharpe = {planted['sharpe']:.2f}")

null (edge=0), 8 seeds: NW t mean -0.30 (sd 0.85), |t|>=2 in 0/8
planted (edge=0.010): NW t = +12.91, Sharpe = 2.35


## Verdict

- **Signal — None.** The claimed global-sovereign-bond time-series-momentum edge does **not** appear. The long-short 12-1 book is flat (**+0.26 bps/mo**, NW *t* = **+0.02**, placebo *p* = 0.72); the long-only +12.27 bps/mo is **bond beta** (under-earns the 24.56 bps/mo buy-and-hold, placebo *p* = 0.53, NW *t* = +1.75 below the bar, fragile across lookbacks). The 20-seed synthetic control fires on a planted trend (mean *t* = +13.38, 0/20 on the null) — the null is real.
- **Tradability — Mirage.** Long-short loses money once costed (net **-3.18 bps/mo** at 5 bps, -4.48 at 10 bps; net Sharpe < 0). Long-only's costed net Sharpe (~0.29) is diluted bond beta that under-earns buy-and-hold — no costed net edge survives.